### PR TITLE
fix dojo cyborg and sushi

### LINKED ISSUE #26065: You can see the ninja den through the cyborg camera.
(WEB REPORT BY: primed REMOTE: 206.221.180.138:7777)
# Revision
bb9be254eab2b999f9a9946613f9d60b0746b520

# Description
The cyborg that is placed in the "Spider Clan Dojo" or whatever has a camera in it. People can see when a ninja spawns using the cyborg camera. 
# Steps to Reproduce
1. Ninja Spawns
2. Crew laughs and watches him practice in the ninja den through the cyborg camera.

### LINKED ISSUE #28022: You can see the borg in the Dojo using the security camera console
(WEB REPORT BY: goodguy1448 REMOTE: 206.221.180.138:7777)
# Revision
134b7338e012e9a778449d8a79a7aa4b926570e7

# Description
You are able to access the Dojo borg's camera using the regular secuirty camera console

# Steps to Reproduce
1- Find or make a security camera console
2- Click the robot icon in the top left
3- Check for default borgs
4- Take a look at the dojo

# What you Expected
Nothing, tbh

# What Actually Happened
Cool place weeb

### LINKED ISSUE #28045: Ninja borg test dummy shows up on security cameras
(WEB REPORT BY: burntdevil REMOTE: 206.221.180.138:7777)
# Revision
b601cfc730820704ad3bd54691eea3be412b66a5

Ya goofed

### PR DESCRIPTION
**Pull request by SonixApache**
[hotfix] closes #28045, closes #28022, closes #26065 
dojo robo is now a subtype with no camera and a crap cell
sushi typepaths fixed
image

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
hooray

File: code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/mob/living/silicon/robot/dummy
+	name = "cybernetic target dummy"
+	desc = "Great for practicing on."
+	cell_type = /obj/item/weapon/cell/crap
+	anchored = TRUE
+	scrambledcodes = TRUE
```
Comment (DamianX) [inline code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm:6]: most of these vars get set by the UnlinkSelf call in New, what's the point of this?

Comment (SonixApache) [reply code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm:6]: i thought i removed the `UnlinkSelf()` call, that proc isn't necessary when these vars are set up right
saves a cpu cycle

Comment (DamianX) [reply code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm:6]: but unlinkself also removes the borg from the camera network

Comment (SonixApache) [reply code/modules/mob/living/silicon/robot/robot_subtypes/dummy.dm:6]: scrambledcodes makes it so it never gets connected to the network in the first place

**Pull request merged**